### ISSUE TITLE
Reader: Clicking Show More when searching "Daring Fireball" doesn't show results

### ISSUE THREAD

**Issue by jancavan** [Automattic/wp-calypso#14297]
#### Steps to reproduce
1. Starting at URL: https://wpcalypso.wordpress.com/following/manage
2. Search for "daringfireball.net"
3. Click on the "Show more" button

#### What happened
I think it's trying to load a few more sites, but it's stuck on the placehoders.

![screenshot 2017-05-19 14 26 30](https://cloud.githubusercontent.com/assets/4924246/26267469/29244314-3c9f-11e7-8ce0-52fe5357b7ad.png)

**Comment by jancavan** [comment]
A similar thing is happening when searching for "thegreatdiscontent.com". Clicking on "Show more" doesn't load anything, although the placeholders aren't shown for this particular search:

https://wpcalypso.wordpress.com/following/manage?q=thegreatdiscontent.com

**Comment by samouri** [comment]
@gibrown, looks like the count is wrong from ES here:
```json
{  
   "code":200,
   "headers":[  
      {  
         "name":"Content-Type",
         "value":"application\/json"
      }
   ],
   "body":{  
      "feeds":{  
         "0":{  
            "URL":"https:\/\/daringfireball.net\/",
            "subscribe_URL":"http:\/\/daringfireball.net\/feeds\/main",
            "feed_ID":"20787116",
            "title":"Daring Fireball",
            "subscribers_count":371,
            "railcar":{  
               "railcar":"A)a0&0*t!3Hm",
               "fetch_algo":"reader\/manage\/search:0",
               "fetch_position":0,
               "rec_feed_id":20787116,
               "fetch_lang":"en",
               "fetch_query":"daringfireball.net"
            },
            "meta":{  
               "links":{  
                  "feed":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/read\/feed\/20787116"
               }
            }
         },
         "4":{  
            "URL":"https:\/\/goldgam.wordpress.com",
            "subscribe_URL":"http:\/\/goldgam.wordpress.com",
            "blog_ID":"27681345",
            "title":"goldgam",
            "railcar":{  
               "railcar":"p0rz0o@00(SM",
               "fetch_algo":"reader\/manage\/search:0",
               "fetch_position":4,
               "rec_blog_id":27681345,
               "fetch_lang":"en",
               "fetch_query":"daringfireball.net"
            },
            "meta":{  
               "links":{  
                  "feed":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/read\/feed\/253163",
                  "site":"https:\/\/public-api.wordpress.com\/rest\/v1.1\/read\/sites\/27681345"
               }
            },
            "feed_ID":"253163",
            "subscribers_count":13
         }
      },
      "total":5
   }
}
```

**Comment by samouri** [comment]
or @blowery could this have to do with all the recent deduping?

**Comment by gibrown** [comment]
Hmmm, that could be related to deduping (ES still had it in the index, but when we tried to get the feed it no longer existed). Similar cases can occur if a site was just blocked or deleted. We could try to adjust the total based on what we know we have removed, but from one call to the next we won't know what we have removed, so we should probably handle the case where the total does not match the items that we returned.
